lib/node/column-namer.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+function stripTableName(name) {
+    return name.split('.').pop();
+}
+
+// This class is used to help generate the names of node query columns
+module.exports = class ColumnNamer {
+    constructor (columns) {
+        this.columns = columns.map(name => stripTableName(name).toLowerCase());
+    }
+
+    uniqueName(baseName) {
+        baseName = baseName.toLowerCase();
+        var name = baseName;
+        var ok = this.isUnique(name);
+        if (!ok) {
```

Comment:
we don't need this initial checking, do we?. I mean if `while(this.isUnique)` is false it skips immediately of 
 suffixing column name, right?

Reply:
Right! (It was needed before I remove the code to use params.id and didn't realize it wasn't needed anymore)